client.lua

```diff
@@ -179,6 +179,30 @@ local function StartFueling(vehicle)
     notify(fuel)
 end
 
+if ox.qtarget then
+    AddEventHandler('ox_fuel:refuelVehicle', function(data)
+        local pumpObject = data.entity
+        local vehicle = GetPlayersLastVehicle()
+
+        if vehicle == 0 or #(GetEntityCoords(vehicle) - GetEntityCoords(pumpObject)) > 3 then
+            return notify('Vehicle far from pump')
+        end
+        StartFueling(vehicle)
+    end)
+
+    exports.qtarget:AddTargetModel(ox.pumpModels, {
```

Comment:
Would probably want to add the export to all of the vehicles instead of all the pump models

Reply:
You mean enum all the vehicles somewhere in ox_fuel resources ? Sounds a pain for servers who use add-on vehicle, no?

Reply:
No I mean using the Vehicle export, adding support for gas cans would be pain otherwise